layer1/wallet/test/unit/test_mint_xtz.mligo

```diff
@@ -18,7 +20,10 @@ let run_main_mint_xtz_test
 
 
   let wallet_initial_storage = {
-    owner_address = Tezos.self_address;
+    // owner_address = (Tezos.get_self_address ());
+    owner_address =
+      (let x = "WARNING: Test.get_self_address is weird here. What I write might be better but not sure" in
```

Comment:
May I know what's your concern?

Reply:
you're right. I forgot why it's Tezos.self_address at the first place, but it's misused. yours is correct.